<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1570</width>
    <height>989</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="device1">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>190</y>
      <width>150</width>
      <height>450</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="device2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>190</y>
      <width>150</width>
      <height>450</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="device3">
    <property name="geometry">
     <rect>
      <x>1250</x>
      <y>190</y>
      <width>150</width>
      <height>450</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="device4">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>190</y>
      <width>150</width>
      <height>450</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="device5">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>190</y>
      <width>150</width>
      <height>450</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>90</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;Jetson 4&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>90</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;Jetson 3&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>1270</x>
      <y>90</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;Jetson 9&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>90</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;Jetson 5&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>90</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;Jetson 6&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="jetson4_button_true">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>750</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Jetson 4 ON</string>
    </property>
   </widget>
   <widget class="QPushButton" name="jetson4_button_false">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>750</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Jetson 4 OFF</string>
    </property>
   </widget>
   <widget class="QPushButton" name="jetson3_button_true">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>750</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Jetson 3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="jetson3_button_false">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>750</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Jetson 3 OFF</string>
    </property>
   </widget>
   <widget class="QPushButton" name="jetson9_button_true">
    <property name="geometry">
     <rect>
      <x>1260</x>
      <y>750</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Jetson 9 ON</string>
    </property>
   </widget>
   <widget class="QPushButton" name="jetson9_button_false">
    <property name="geometry">
     <rect>
      <x>1260</x>
      <y>750</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Jetson 9 OFF</string>
    </property>
   </widget>
   <widget class="QPushButton" name="jetson5_button_true">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>750</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Jetson 5 ON</string>
    </property>
   </widget>
   <widget class="QPushButton" name="jetson5_button_false">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>750</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Jetson 5 OFF</string>
    </property>
   </widget>
   <widget class="QPushButton" name="jetson6_button_true">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>750</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Jetson 6 ON</string>
    </property>
   </widget>
   <widget class="QPushButton" name="jetson6_button_false">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>750</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Jetson 6 OFF</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1570</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
